src/Components/Cart/Cart.css

```diff
@@ -1,16 +1,29 @@
-/* .cart-container{
```

Comment:
Overall everything looks good. Your code is clean. Few suggestions are mentioned below. Please check. Apart from them , you may want to display the price under Slider as a range (eg Rs 0 to Rs 100) rather than just a number for better UI.

Reply:
OK i will work on that also